File: Polimorfismo - Batalha Final/Chefe.java
public class Chefe extends Personagem{
    
    public Chefe(String nome){
        super(nome, 200, 20, 1);
    }
    @Override
    public String habilidade(){
        return "ataque devastador";
    }
    @Override
    public void ficha(){
        System.out.println("\n[CHEFE] " + getNome() + " (vida: " + getVida() + ",  forca: " + getForca() +")");
    }
}

File: Exceções - IF Quest/chefe.py
from personagem import Personagem

class Chefe(Personagem):
    def __init__(self, n="Chefe"):
        super().__init__(n, 200, 20, 1)
    def habilidade(self):
        return "ataque devastador"
    def ficha(self):
        print("\n[Chefe] ", self._nome, " Força: ", self._forca, " Vida: ", self._vida)
